core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerImpl.java

```diff
@@ -175,11 +108,25 @@ private void requestSurface(SurfaceHolderRequester.Callback callback) {
         surfaceHolderRequester.requestSurfaceHolder(callback);
     }
 
+    void resetSeek() {
```

Comment:
It won't be used by any of the clients of `Player`, and very likely not be needed for the other implementation.